portal/pom.xml

```diff
@@ -41,18 +41,48 @@
       <version>${project.version}</version>
       <scope>runtime</scope>
     </dependency>
+    <dependency>
+      <groupId>org.apache.tomcat</groupId>
+      <artifactId>tomcat-catalina</artifactId>
+      <version>7.0.68</version>
+      <scope>runtime</scope>
```

Comment:
Including tomcat-catalina dependency broke the deployment to tomcat. 

http://stackoverflow.com/questions/14028476/java-web-api-conflict-using-maven

I'll make a quick fix to set the scope to provided.